### ISSUE TITLE
removeUnusedAtRules doesn't work when there are multiple rules per font

### ISSUE THREAD

**Issue by clarcharr** [jakubpawlowicz/clean-css#950]
Minimal example:

```
@font-face {
    font-family: font;
    font-weight: regular;
}
@font-face {
    font-family: font;
    font-weight: bold;
}
```

This will minify to:

```
@font-face{font-family:font;font-weight:regular}
```

Which is wrong.

**Comment by clarcharr** [comment]
It appears to be an error in this function: https://github.com/jakubpawlowicz/clean-css/blob/f273443f3d697764777e4618121d780c6894a922/lib/optimizer/level-2/remove-unused-at-rules.js#L112

The check for used `font-family` rules doesn't check if the block is itself a `@font-face` block.

**Comment by jakubpawlowicz** [comment]
👍 it should be checked that way indeed, thanks for digging for the solution

**Comment by jakubpawlowicz** [comment]
Fixed in 4.1.4!

**Comment by clarcharr** [comment]
Thank you!
